libs/utils/utils.js

```diff
@@ -483,15 +483,23 @@ export function decorateImageLinks(el) {
   const images = el.querySelectorAll('img[alt*="|"]');
   if (!images.length) return;
   [...images].forEach((img) => {
-    const [source, alt] = img.alt.split('|');
+    const [source, alt, playBtn] = img.alt.split('|');
     try {
       const url = new URL(source.trim());
       if (alt?.trim().length) img.alt = alt.trim();
       const pic = img.closest('picture');
       const picParent = pic.parentElement;
       const aTag = createTag('a', { href: url, class: 'image-link' });
       picParent.insertBefore(aTag, pic);
-      aTag.append(pic);
+      if (playBtn?.includes(':play')) {
+        const initVideoModal = async () => {
+          const { default: init } = await import('./imageVideoLink.js');
+          init(picParent, aTag, playBtn);
+        };
+        initVideoModal();
+      } else {
+        aTag.append(pic);
+      }
```

Comment:
```suggestion
      if (icon) {
        import('./image-video-link.js').then((mod) => mod.default(picParent, aTag, icon));
      } else {
        aTag.append(pic);
      }
```
I don't think we need to care what icon they want for the button.

Reply:
@auniverseaway I have made the suggested changes.